### PR TITLE
Refactor flytecopilot container command

### PR DESCRIPTION
**Pull request by evalsocket**
# TL;DR

Context:- Currently copiliot share the config of flytepropeller but we don't want any limit on upload for copiliot. 
Solution:- 
- Added command line flag in flytecopilot for storage config
- Added new image of copiliot
- 
## Type
 - [ ] Bug Fix
 - [x] Feature
 - [ ] Plugin

## Are all requirements met?

 - [ ] Code completed
 - [ ] Smoke tested
 - [ ] Unit tests added
 - [ ] Code documentation added
 - [ ] Any pending items have an associated Issue

## Complete description
 _How did you fix the bug, make the feature etc. Link to any design docs etc_

## Tracking Issue

fixes https://github.com/flyteorg/flyte/issues/1251

## Follow-up issue
_NA_
OR
_https://github.com/flyteorg/flyte/issues/<number>_

### CODE REVIEW

File: go/tasks/pluginmachinery/flytek8s/copilot.go [hunk 1]
```diff
@@ -43,7 +43,7 @@ func FlyteCoPilotContainer(name string, cfg config.FlyteCoPilotConfig, args []st
 	return v1.Container{
 		Name:       cfg.NamePrefix + name,
 		Image:      cfg.Image,
-		Command:    []string{"/bin/flyte-copilot", "--config", "/etc/flyte/config**/*"},
+		Command:    CopilotCommandArgs(storage.GetConfig()),
```
Comment (kumare3) [inline go/tasks/pluginmachinery/flytek8s/copilot.go:46]: looks good to me, can we update the default copilot image in flyteplugins. [This one](https://github.com/flyteorg/flyteplugins/blob/master/go/tasks/pluginmachinery/flytek8s/config/config.go#L30) should match the one that contains the latest flytestdlib -

Comment (kumare3) [reply go/tasks/pluginmachinery/flytek8s/copilot.go:46]: lgtm after taht

File: go/tasks/pluginmachinery/flytek8s/config/config.go [hunk 2]
```diff
@@ -27,7 +27,7 @@ var (
 		},
 		CoPilot: FlyteCoPilotConfig{
 			NamePrefix:           "flyte-copilot-",
-			Image:                "docker.pkg.github.com/flyteorg/flyteplugins/operator:v0.4.0",
+			Image:                "ghcr.io/flyteorg/flytecopilot:v0.0.9",
```
Comment (EngHabu) [inline go/tasks/pluginmachinery/flytek8s/config/config.go:30]:
```suggestion
			Image:                "cr.flyte.org/flyteorg/flytecopilot:v0.0.9",
```

File: go/tasks/pluginmachinery/flytek8s/copilot.go [hunk 3]
```diff
@@ -62,6 +63,34 @@ func FlyteCoPilotContainer(name string, cfg config.FlyteCoPilotConfig, args []st
...
+		"--storage.enable-multicontainer",
+		"--storage.limits.maxDownloadMBs=0",
+		fmt.Sprintf("--storage.type=%s", storageConfig.Type),
+	}
+	if !reflect.DeepEqual(storageConfig.Stow, storage.StowConfig{}) {
+		var cfg string
+		for key, val := range storageConfig.Stow.Config {
+			cfg += fmt.Sprintf("%s=%s,", key, val)
+		}
+		return append(commands, []string{
```
Comment (EngHabu) [inline go/tasks/pluginmachinery/flytek8s/copilot.go:78]: should we end here? I think we should append all the options and let the storage library decide which ones take over the others?

Comment (evalsocket) [reply go/tasks/pluginmachinery/flytek8s/copilot.go]: that true...I added the changes

**Review by EngHabu — APPROVED** [review]
